automation/jleechanorg_pr_automation/openai_automation/oracle_cli.py

```
@@ -101,10 +123,12 @@ async def navigate_to_chatgpt(self):
                 timeout=5000
             )
             print("✅ Logged in to ChatGPT")
+            await self._save_auth_state()
         except PlaywrightTimeoutError:
             print("⚠️  Not logged in - please log in manually")
             print("   Waiting 30 seconds for you to log in...")
             await asyncio.sleep(30)
+            await self._save_auth_state()
         except Exception as login_error:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)